client/webserver/site/src/js/order.ts

```diff
@@ -567,6 +567,10 @@ const CoinExplorers: Record<number, Record<number, (cid: string) => string>> = {
     [Mainnet]: (cid: string) => `https://dogeblocks.com/tx/${cid.split(':')[0]}`,
     [Testnet]: (cid: string) => `https://blockexplorer.one/dogecoin/testnet/tx/${cid.split(':')[0]}`
   },
+  136: { // firo
+    [Mainnet]: (cid: string) => `https://explorer.firo.org/tx/${cid.split(':')[0]}`,
+    [Testnet]: (cid: string) => `https://testexplorer.firo.org/tx/${cid.split(':')[0]}`
```

Comment:
https://testexplorer.firo.org/ seesm to be stuck for a few days.  Any way to ping the admins?

Reply:
At the moment you can just cpu mine testnet. 
Anwar in https://t.me/firosupport can gpu mine it when it gets busy. UTC+8 is his timezone (as is mine)

Reply:
The Firo community should keep it moving

Reply:
Yes they should .. actually will get busier this month with some release candidates of Lelantus Spark protocol which community will help test.
There is also a semi-private `devnet` where most of the zkp changes get developed  which is more used than this testnet. 

For me I could generate blocks on testnet easily this week. 
`./firod -testnet -daemon`
...
`./firo-cli -testnet generate 1` 
will move the explorer 1 block
Sync: This testnet current size is maybe 200MB

and if want coins there is a testnet faucet at:
https://testexplorer.firo.org/faucet
..with 200k coins